src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should be case-insensitive', () => {
+    expect(isIsogram('isisOGraM')).toBe(false);
+  });
+
+  it('Word without duplicated character', () => {
+    expect(isIsogram('twelve')).toBe(false);
+  });
+
+  it('Word with duplicated character in mixed case', () => {
+    expect(isIsogram('Alphabet')).toBe(false);
```

Comment:
The word 'Alphabet' is an isogram because all the letters appear only once, and the function should be case-insensitive. The test should expect true, not false for this case.